### ISSUE TITLE
Content audit of cloud.gov technology and security pages

### ISSUE THREAD

**Issue by karareinsel** [cloud-gov/cg-product#1302]
As a cloud.gov team member, I would like to audit the technology and security content on cloud.gov to ensure the content is relevant and up to date for prospective and existing customers and development team members. The audit will assess if content should be preserved, consolidate/edited, or sunsetted.

AC

- [x] Proposed changes are documented

Site IA will be updated via a separate story in a future sprint.

Link to audit template - https://docs.google.com/spreadsheets/d/1zaRnJfUr9J3K2LcasMHs9PRD9KNxFiT1pONXqbc7v20/edit#gid=0

Link to site - https://cg-4bc824cf-0e89-4d31-a1be-d64b4c9fec53.app.cloud.gov/preview/18f/cg-site-federalist/colors-and-content/

Link to current sitemap - https://app.mural.co/t/gsa6/m/gsa6/1581345078867/ecddcf20b48d92e1b5493ecbf0a78860f2a123eb

Link to Google Analytics data - https://docs.google.com/spreadsheets/d/1GlbuT-hgS27-Wqui1QuBhlHp1yWccdcj6jClOQ3lUEo/edit#gid=1470290569

**Comment by karareinsel** [comment]
Content changes will be implemented once #1557 is complete.

**Comment by karareinsel** [comment]
Changes have been documented and will be implemented once the Netlify CMS has been added.
